packages/go_router/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.0.3
+
+- Fixed a bug where params disappear when pushing a nested route.
```

Comment:
Reviewers: For future reviews, please make sure that CHANGELOG entries follow [the repository style guide](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style). (This should have been "Fixes", not "Fixed".)